tests/gemm.cpp

```diff
@@ -369,4 +370,394 @@ TEST(BLAS3_trmm, left_lower_tri_implicit_diag)
     }
   }
 
+TEST(BLAS3_trmm, left_upper_tri_explicit_diag)
```

Comment:
I do appreciate all the new tests -- thanks! : - D

Reply:
Untested code is broken code :-)